libs/core/src/lib/multi-combobox/select-all-toggler.component.ts

```diff
@@ -0,0 +1,72 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, inject, ViewEncapsulation } from '@angular/core';
+import { DestroyedService } from '@fundamental-ngx/cdk/utils';
+import { takeUntil } from 'rxjs';
+import { MultiComboboxComponent } from './multi-combobox.component';
+
+@Component({
+    selector: 'fd-multi-combobox-select-all-toggler',
+    template: `
+        <fd-checkbox
+            [label]="'Select All'"
+            [ngModel]="checkboxValue"
+            (ngModelChange)="change($event)"
+            [tristate]="true"
+        ></fd-checkbox>
+    `,
+    styles: [
+        `
+            fd-multi-combobox-select-all-toggler.fd-list__item {
```

Comment:
Should be a fundamental-styles thing too I think?

Reply:
I could not find anything in wiki about this, only at experience sap, so I took and did it only here because of that